__tests__/integration/verifyTemplateTitle.test.js

```diff
@@ -12,6 +12,6 @@ describe('Verify the resource template title is displayed in the editor', () =>
     expect.assertions(2)
 
     await pupExpect(page).toClick('a', { text: 'Note' })
-    await pupExpect(page).toMatchElement('h1', { text: 'Note' })
+    await pupExpect(page).toMatchElement('#resourceTemplate h1', { text: 'Note' })
```

Comment:
selector is more specific, and also is readable as more "aboutness" in the selector.